AutoClicker/Views/MainWindow.xaml

```diff
@@ -122,16 +122,24 @@
                             </Grid.ColumnDefinitions>
 
                             <TextBox Grid.Column="0" Width="45"
-                                     Text="{Binding AutoClickerSettings.MaximumHours, UpdateSourceTrigger=PropertyChanged, Mode=TwoWay}"/>
+                                     Text="{Binding AutoClickerSettings.MaximumHours, UpdateSourceTrigger=PropertyChanged, Mode=TwoWay}"
+                                     IsEnabled="{Binding AutoClickerSettings.IsRandomizedIntervalEnabled, UpdateSourceTrigger=PropertyChanged, Mode=TwoWay}"

```

Comment:
@bnn678 Also consider using "IsReadOnly" instead of "IsEnabled"

Reply:
What is the difference between the two?

Reply:
In the context of a TextBox, IsReadOnly allows the user to set focus to and select and copy the text, but not modify it.
A disabled TextBox does not allow any interaction whatsoever.

As a general rule, use IsReadOnly when you have data that you want the user to see and copy, but not modify.
Use a disabled textbox when the data you are displaying is not applicable for the current state of a dialog or window.

Reply:
I'm leaning toward rescoping this ticket and removing the textbox disabling since I can't the binding to update. Then a later PR can disable the textboxes when appropriate.